src/ui/SolidityState.js

```diff
@@ -44,11 +44,13 @@ SolidityState.prototype.init = function () {
 
     self.traceManager.getStorageAt(index, this.parent.tx, function (error, storage) {
       if (error) {
-        self.basicPanel.update({ info: error })
+        self.basicPanel.update({})
```

Comment:
yes that just a temp fix.
I will open an issue for that, we need to better report error to user